experiment/measurer.py

```diff
@@ -181,7 +70,9 @@ def measure_main(experiment_config):
     measure_loop(experiment, max_total_time)
 
     # Do the final measuring and store the coverage data.
-    store_coverage_data(experiment_config)
+    cov_utils.store_coverage_data(experiment_config)
+    cov_utils.generate_cov_reports(experiment_config)
```

Comment:
use full function names, s/generate_cov_reports /generate_coverage_report
and next one s/update_coverage_to_bucket/upload_coverage_report_to_bucket